Program.cs

```diff
@@ -0,0 +1,322 @@
+            {
+                Console.WriteLine(question);
+                response = Console.ReadLine();
+            }
+
+            return response;
+        }
+        static string AskAboutYear(string question)
+        {
+            string response = "";
+            while (response == "")
+            {
+                Console.WriteLine(question);
+                response = Console.ReadLine();
+            }
+            return response;
+        }
+        static string AskAboutColor(string question)
+        {
+            string response = "";
+            while (response == "")
+            {
+                Console.WriteLine(question);
+                response = Console.ReadLine();
+            }
+            return response;
+        }
+        static string AskAboutAnimal(string question)
+        {
+            string response = "";
+            while (response == "")
+            {
+                Console.WriteLine(question);
+                response = Console.ReadLine();
+            }
+            return response;
+        }
+        static string AskHowManyDogs(string question)
+        {
+            string response = "";
+            while (response == "")
+            {
+                Console.WriteLine(question);
+                response = Console.ReadLine();
+            }
+            return response;
+        }
+        static string AskHowManyCats(string question)
+        {
+            string response = "";
+            while (response == "")
+            {
+                Console.WriteLine(question);
+                response = Console.ReadLine();
+            }
+            return response;
+        }
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Hi, my name is Macy.");
+            System.Threading.Thread.Sleep(1000);
+            Console.WriteLine("I think we could be friends");
+            System.Threading.Thread.Sleep(1000);
+            Console.WriteLine("What is your first name?");
+            string firstName = Console.ReadLine();
+            Console.WriteLine("and your last name?");
+            string lastName = Console.ReadLine();
+            Console.WriteLine("Dont worry I am not going to Facebook stock you......");
+            System.Threading.Thread.Sleep(1000);
+            Console.WriteLine("Is it okay if I ask you a couple of questions " + firstName + "?");
+            string okayToAsk = Console.ReadLine();
+            if (okayToAsk == "no")
+            {
+                int numberOfTimesToAsk = 1;
+                do
+                {
+                    Console.WriteLine("Thats fine. I guess..... its a big world!");
+                    System.Threading.Thread.Sleep(1000);
+                    Console.WriteLine("I will just have to find another friend");
+                    System.Threading.Thread.Sleep(1000);
+                    Console.WriteLine("Some where else!");
+                    System.Threading.Thread.Sleep(1000);
```

Comment:
This pattern of displaying/pausing is repeated throughout. Perhaps a helper method could be written such as:
```CS
static void WriteWithPause(string message)
{
  Console.WriteLine(message);
  System.Threading.Thread.Sleep(1000);
}
```
Or, if you might pause for different amounts at different times in the app:
```CS
static void WriteWithPause(string message, int millisecondsToPause)
{
  Console.WriteLine(message);
  System.Threading.Thread.Sleep(millisecondsToPause);
}
```